dashboards/persistentvolumesusage.libsonnet

```
@@ -0,0 +1,110 @@
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local promgrafonnet = import '../lib/promgrafonnet/promgrafonnet.libsonnet';
+local numbersinglestat = promgrafonnet.numbersinglestat;
+local gauge = promgrafonnet.gauge;
+
+{
+  grafanaDashboards+:: {
+    'persistentvolumesusage.json':
+      local sizeGraph = graphPanel.new(
+        'Volume size usage',
```

```suggestion
        'Volume Space Usage',
```
To be consistent with the Nodes dashboard.